d2bs/kolbot/libs/bots/ControlBot.js

```diff
@@ -410,43 +390,35 @@ function ControlBot() {
 			break;
 		}
 
-		let wpList;
 		let act = Misc.getPlayerAct(nick);
-
-		switch (act) {
-		case 1:
-			wpList = [
-				sdk.areas.ColdPlains, sdk.areas.StonyField, sdk.areas.DarkWood, sdk.areas.BlackMarsh,
-				sdk.areas.OuterCloister, sdk.areas.JailLvl1, sdk.areas.InnerCloister, sdk.areas.CatacombsLvl2
-			];
-
-			break;
-		case 2:
-			wpList = [
-				sdk.areas.A2SewersLvl2, sdk.areas.DryHills, sdk.areas.HallsoftheDeadLvl2, sdk.areas.FarOasis,
-				sdk.areas.LostCity, sdk.areas.PalaceCellarLvl1, sdk.areas.ArcaneSanctuary, sdk.areas.CanyonofMagic
-			];
-
-			break;
-		case 3:
-			wpList = [
-				sdk.areas.SpiderForest, sdk.areas.GreatMarsh, sdk.areas.FlayerJungle, sdk.areas.LowerKurast,
-				sdk.areas.KurastBazaar, sdk.areas.UpperKurast, sdk.areas.Travincal, sdk.areas.DuranceofHateLvl2
-			];
-
-			break;
-		case 4:
-			wpList = [sdk.areas.CityoftheDamned, sdk.areas.RiverofFlame];
-
-			break;
-		case 5:
-			wpList = [
-				sdk.areas.FrigidHighlands, sdk.areas.ArreatPlateau, sdk.areas.CrystalizedPassage,
-				sdk.areas.GlacialTrail, sdk.areas.FrozenTundra, sdk.areas.AncientsWay, sdk.areas.WorldstoneLvl2
-			];
-
-			break;
-		}
+		let wpList = (() => {
```

Comment:
this would be more appropriate as an obj or even array:
```
let wps = {
    1: [1, 2, 3, ... , n],
    2: [1, 2, 3, ... , n]
    3: [1, 2, 3, ... , n]
    ...
    n: [1, 2, 3, ... , n]
};

return wps[act]

let wps = [
    [1, 2, 3, ... , n],
    [1, 2, 3, ... , n],
    [1, 2, 3, ... , n],
    ...
    [1, 2, 3, ... , n],
]

return wps[act - 1]
```